options/base_options.py

```diff
@@ -85,6 +85,12 @@ def initialize(self, parser):
         parser.add_argument('--D_label_smooth', action='store_true', help='whether to use one-sided label smoothing with discriminator')
         parser.add_argument('--D_noise', type=float, default=0.0, help='whether to add instance noise to discriminator inputs')
 
+        #Online dataset creation options
+        parser.add_argument('--online_creation_crop_size', type=int, default=512, help='then crop to this size during online creation, it needs to be greater than bbox dim')
+        parser.add_argument('--online_creation_crop_delta', type=int, default=50, help='delta around the crops dim')
```

Comment:
Documentation not clear, as I understood from the code the delta can be random? and what does "around the crops dim" means?